src/main/java/bio/terra/cli/command/user/sshkey/Get.java

```diff
@@ -0,0 +1,27 @@
+package bio.terra.cli.command.user.sshkey;
+
+import bio.terra.cli.command.shared.BaseCommand;
+import bio.terra.cli.command.shared.options.Format;
+import bio.terra.cli.serialization.userfacing.UFSshKey;
+import bio.terra.cli.service.ExternalCredentialsManagerService;
+import bio.terra.externalcreds.model.SshKeyPairType;
+import picocli.CommandLine;
+import picocli.CommandLine.Command;
+
+@Command(name = "get", description = "Get a terra generated and managed ssh key.")
```

Comment:
This is actually "get, or create if it doesn't exist" which I wouldn't expect from the command `terra user sshkey get`. 
Should `get and `create` be different commands, or is there a way to make this command's function clearer?

Reply:
nit: "terra-generated and -managed"

Reply:
I'm not opposed to having a create but because the UI auto create one, this is suggested by Jenni.